src/main/java/io/github/crew102/rapidrake/RakeAlgorithm.java

```diff
@@ -64,6 +64,36 @@ public RakeAlgorithm(RakeParams rakeParams, InputStream taggerStream, InputStrea
     this.tagger = new Tagger(taggerStream).getPosTagger();
     this.sentDetector = new SentDetector(sentDectStream).getSentDetector();
   }
+
+  /**
+   * Constructor.
+   *
+   * @param rakeParams the parameters RAKE will use
+   * @param tagger An instance of opennlp.tools.postag.POSTaggerME.
+   * @param sentDetector An instance of opennlp.tools.sentdetect.SentenceDetectorME.
+   * @see RakeParams
+   * @throws java.io.IOException if either of the input streams are invalid
+   */
+  public RakeAlgorithm(RakeParams rakeParams, POSTaggerME posTaggerMEInstance, SentenceDetectorME sentDetectorMEInstance) throws java.io.IOException {
+    this.rakeParams = rakeParams;
+    this.tagger = posTaggerMEInstance;
+    this.sentDetector = sentDetectorMEInstance;
+  }
+
+  /**
+   * Constructor.
+   *
+   * @param rakeParams the parameters RAKE will use
+   * @param taggerInstance An instance of io.github.crew102.rapidrake.opennlpUtils.Tagger.
+   * @param sentDectInstance An instance of io.github.crew102.rapidrake.opennlpUtils.SentDetector.
+   * @see RakeParams
+   * @throws java.io.IOException if either of the input streams are invalid
+   */
+  public RakeAlgorithm(RakeParams rakeParams, Tagger taggerInstance, SentDetector sentDectInstance) throws java.io.IOException {
```

Comment:
What's the use case for this constructor, given the `getPosTagger()` call is what takes the time?

Reply:
That constructor really doesn’t offer anything. I implemented that
originally to see if that offered any benefits, but it does not.

I should have just removed it.